paasta_tools/instance/kubernetes.py

```diff
@@ -826,7 +826,7 @@ async def get_pod_containers(
                     reason = this_state["reason"]
                 if "message" in this_state:
                     message = this_state["message"]
-                if "started_at" in this_state:
+                if this_state.get("started_at"):
                     start_timestamp = this_state["started_at"].timestamp()
```

Comment:
If `started_at` is `None`, then there would be an exception when calling `.timestamp()` I think. You'll also need to check for the non-null case, but I'm not sure what you would retrieve here. Just set `this_state["started_at"]` to `None` maybe?

Reply:
if started_at is None or empty or otherwise False-y, we won't get to the point where we call `this_state["started_at"].timestamp()`. That's exactly the situation we are trying to fix here.  

I figured we don't really need to distinguish from the state not having this info at all to having unuseful/blank info.